get.go

```diff
@@ -14,13 +14,27 @@ import (
 )
 
 func getNewestMasterTag(tagList []string) string {
-
 	for _, tag := range tagList {
-		if strings.Index(tag, "master") != -1 {
-			return tag
+		if strings.Index(tag, "master-") != -1 {
+			if isUnique(tag, tagList) {
```

Comment:
@koudaiii 's comment means that "Is it really necessary to check the uniqueness of `tagList`?"

I think that tags attached one image are completely unique, so we do not have to check uniqueness again here.